driver/controller.go

```diff
@@ -33,11 +33,11 @@ import (
 )
 
 const (
-	_  = iota
-	KB = 1 << (10 * iota)
-	MB
-	GB
-	TB
+	_   = iota
+	kiB = 1 << (10 * iota)
```

Comment:
An alternative is to un-export these constants since they're not used outside of this package anyway. I added a commit to that effect.

If you prefer the exported constants without godoc, just let me know and I'll make that change.